### DISCUSSION TITLE
Docker.sock permissions? incorrect volume name?

### DISCUSSION THREAD

**Discussion by Masterism88** [nextcloud/all-in-one#2099] [Questions] [answered]
So, I tried to install the nextcloud aio mastercontainer on unraid and for reasons I had to do a reset. I'm now trying to install it again and I'm getting this error `Trying to fix docker.sock permissions internally...
Creating docker group internally with id 281
It seems like you did not give the mastercontainer volume the correct name?
Using a different name is not supported!` Nothing has changed, I followed [These](https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance) instructions and did exactly what it said to do. Any ideas on what I can do?  Thanks

**Comment by szaimen** [comment, marked as answer]
Lets handle this in https://forums.unraid.net/topic/135674-nextcloud-aio-database-bootloop
